### ISSUE TITLE
ZGW Configuration doesn't work

### ISSUE THREAD

**Issue by sergei-maertens** [open-formulieren/open-forms#521]
When I select a zaaktype from the dropdown, I get a validation error that <url> is not a valid choice. This happens always.

URL: https://open-forms.test.maykin.opengem.nl/admin/zgw_apis/zgwconfig/

**Comment by Bartvaderkin** [comment]
This check is probably incorrect: https://github.com/open-formulieren/open-forms/blob/89e94fc71efe00047d60c6a9cc7840c887618397/src/openforms/registrations/contrib/zgw_apis/models.py#L72-L86

**Comment by sergei-maertens** [comment]
Can't reproduce atm in test environment, but this might be related to uwsgi processes -> try and reproduced with docker (compose) locally.

**Comment by shea-maykinmedia** [comment]
Tested this locally, both with docker and Django's regular runserver, but I couldn't reproduce this.  Also double checked saving all the possible options but they all worked.

As Bart mentioned if the `zaaktype` (which is a url) doesn't start with the API root of the Catalogi API (ztc_service) this would throw an exception but I couldn't replicate this happening so I'm not sure if or when this would happen.

Going to close this now as I've spent the discussed timebox time limit on it.
